### DISCUSSION TITLE
Implement Chat bot

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11487] [Q&A]
How to implement the chatbot in orchard core decouple cms easily, please advise

**Comment by hishamco** [comment]
Simple answer try to find Bot SDK that's integrated with ASP.NET Core and you are free to integrate with it

**Comment by ns8482e** [comment]
Search google

**Comment by Skrypt** [comment]
We implemented the Facebook Messenger Chat who's fairly easy to implement. It is just javascript code that you need to add to your website frontend theme or specific View.

**Comment by saleemhaq** [comment]
Can you please share with me the sample code?

**Comment by saleemhaq** [comment]
You have any experience regarding integration of bot if any so please let
me know to achieve end to end working.

**Reply by ns8482e** [reply]
Do you have any specific issue  that you are facing integrating chat bot with OchardCore ? But works in regular Asp.net application?
